Update documentation

andreas-bok-sociomantic opened the pull request:
Adds documentation for v5.4.0 API and previously omitted functions

Minor formatting fixes

andreas-bok-sociomantic committed:
Update documentation

Adds documentation for the functions:

- drizzle_column_type_str
- drizzle_strerror

andreas-bok-sociomantic committed:
Documentation formatting fixes

gautam-kotian-sociomantic commented on docs/api/connection.rst:
```diff
@@ -204,6 +204,21 @@ Functions
+   Gets the owner of the socket connection
+
+   :param options: The options object to get the value from
+   :returns: The owner of the socket
```
You seem to always say "owner of the socket connection", so it seems odd that the word `connection` is missing here.

andreas-bok-sociomantic replied:
Will change.

gautam-kotian-sociomantic commented on docs/api/constants.rst:
```diff
@@ -730,6 +730,17 @@ Connection
       :py:const:`DRIZZLE_CAPABILITIES_PROTOCOL_41`,
       :py:const:`DRIZZLE_CAPABILITIES_SECURE_CONNECTION`
 
+.. c:type:: drizzle_ssl_state_t
```
The commit message doesn't mention anything about adding documentation for the enum. Do you want to add that, or do you want to roll these changes into a separate commit?

andreas-bok-sociomantic replied:
Yeah. Would make sense with a separate commit.

andreas-bok-sociomantic committed:
Update documentation for v5.4.0 API

Adds documentation for changes added in #74

Relates to the functions:
- drizzle_options_set_socket_owner
- drizzle_options_get_socket_owner
and the new enum:
 - drizzle_socket_owner

andreas-bok-sociomantic commented:
Updated based on comments from @gautam-kotian-sociomantic 
```diff
diff --git a/docs/api/connection.rst b/docs/api/connection.rst
index 4fd05c7..74e6787 100644
--- a/docs/api/connection.rst
+++ b/docs/api/connection.rst
@@ -217,7 +217,7 @@ Functions
    Gets the owner of the socket connection
 
    :param options: The options object to get the value from
-   :returns: The owner of the socket
+   :returns: The owner of the socket connection
 ```

andreas-bok-sociomantic commented:
More comments?

gautam-kotian-sociomantic commented:
LGTM.

andreas-bok-sociomantic commented:
Allright, feel free to merge
